Controller/AuthorizeController.php

```diff
@@ -152,7 +152,8 @@ protected function getClient()
             if (null !== $request) {
                 if (null === $clientId = $request->get('client_id')) {
                     $form = $this->container->get('fos_oauth_server.authorize.form');
```

Comment:
In Symfony 3, **deep** argument for `$request->get` has been removed. [ref: method_get symfony api](http://api.symfony.com/3.0/Symfony/Component/HttpFoundation/ParameterBag.html#method_get)
so `$request->get('fos_oauth_server_authorize_form[client_id]'))`  returns `null`

i am not sure what would be the right way to fix it. 

i can think of two ways, 

1.) 
```
 $formData = $request->get($form->getName(), []);
 $clientId = isset($formData['client_id']) ? $formData['client_id'] : null;
```

or 
2.) 
```
$formName = $this->container->get('fos_oauth_server.authorize.form')->getName();
 if (!$request->query->all() && $request->request->has($formName)) {
            $request->query->add($request->request->get($formName));
 }
```